### PR TITLE
CVE-2011-0762 VSFTPD DOS attack

### PR DESCRIPTION
**Pull request by rad10**
This is an auxiliary for DOSing a VSFTPD server from version 2.3.3 and below.

## Verification

List the steps needed to make sure this thing works

- [ ] Start `msfconsole`
- [ ] `use auxiliary/dos/ftp/vstfpd_232`
- [ ] `set rhosts`
- [ ] `set ftpuser`
- [ ] `set ftppass`
- [ ] `run`

### CODE REVIEW

**Comment by rad10** [comment]
I wrote this exploit based off of the POC in https://www.exploit-db.com/exploits/16270. The only problem with this exploit is that it needs multiple connections to run. I got it to work with at least 3 connections, but I'm not sure how I can make multiple separate connections to the server and transmit the payload. Any suggestions would be appreciated.

File: modules/auxiliary/dos/ftp/vstfpd_232.rb [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+        'Author' => [ 'Nick Cottrell <ncottrellweb[at]gmail.com>', 'Anna Graterol <annagraterol95[at]gmail.com>', 'Mana Mostaani <mana.mostaani[at]gmail.com>' ],
+        'License' => MSF_LICENSE,
+        'References' => [
+          [ 'BID', '46617' ],
+          [ 'CVE', '2011-0762' ],
+          [ 'EDB', '16270' ]
+        ],
+        'DisclosureDate' => '2011-02-03',
+        'Notes' => {
+          'Stability' => [CRASH_SAFE],
```
Comment (h00die) [inline modules/auxiliary/dos/ftp/vstfpd_232.rb:27]: A crash safe dos?

Comment (gwillcox-r7) [reply modules/auxiliary/dos/ftp/vstfpd_232.rb:27]: You can use an empty value for this so it would look like the following:

`'Stability' => [],`

**Comment by gwillcox-r7** [comment]
Atticing this until docs can be provided.

**Pull request closed without merging**